### PR TITLE
Test in v0.4.7 also.

### PR DESCRIPTION
**Pull request by Ismael-VC**
Follow up of https://github.com/Ismael-VC/Suppressor.jl/pull/6, this PR adds the v0.4.7 to the matrix configurations of both AppVeyor and Travis CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ os:
   - linux
   - osx
 julia:
+  - 0.4.7
```
Comment (ssfrr) [inline .travis.yml:7]: Should this be `0.4` so it'll keep getting tested on future point releases of 0.4?

File: appveyor.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 environment:
   matrix:
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.4/julia-0.4.7-win32.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x64/0.4/julia-0.4.7-win64.exe"
```
Comment (ssfrr) [inline appveyor.yml:4]: This could also be julia-0.4-latest

Comment (Ismael-VC) [reply appveyor.yml:4]: Done in #8 :+1:

File: .travis.yml (hunk 1 above)
Comment (Ismael-VC) [reply .travis.yml:7]: Done, thanks for your input ...next documentation! :smile:

**Pull request merged**